### ISSUE TITLE
Article Page

### ISSUE THREAD

**Issue by antoniolofiego** [100DaysOfCloud/100DaysOfCloudFrontend#8]
Hosting of #100DaysOfCloud published articles written for the website.

**Comment by andreacavagna01** [comment]
There is a template for this?
I didn't found one on Figma, need some advice? I can use my template blog templates if needed.

**Comment by Bruck1701** [comment]
you mean the code for the page? 
I believe you could use [Figma API](https://www.figma.com/blog/introducing-figma-to-react/) to generate React code from the design. I haven't used it yet, though ...

![articlePage](https://user-images.githubusercontent.com/17711277/88801279-bab00f00-d1a9-11ea-8c5e-09048e59bc71.jpg)

**Comment by andreacavagna01** [comment]
Yeah this, If no one is on this page, I can start Developing this

**Comment by Bruck1701** [comment]
Cool! I don't know if the API to fetch the posts is ready, though..
(Edit) 
I just saw on the Backend repo that @antoniolofiego has mentioned to use [GraphQL](https://github.com/100DaysOfCloud/100DaysOfCloudBackend/issues/31)

**Comment by antoniolofiego** [comment]
The API is not ready yet but the components could be built out of mocked API calls. Also, GraphQL is most likely a future feature that we don't plan to have for the current v1.0 as it adds a lot of complexity. If @andreacavagna01 has any suggestions on how to handle the backend portion of hosted articles and what API design might be best, please feel free to share! Currently, the planned articles APIs only handle redirect articles and not hosted ones.

**Comment by antoniolofiego** [comment]
@andreacavagna01 any update on this?

**Comment by andreacavagna01** [comment]
Tomorrow I will check, sorry I'm busy because tomorrow I will have Solution Architect Exam

**Comment by antoniolofiego** [comment]
No problem at all! Good luck with that 🚀☁
